### ISSUE TITLE
Tronxy XY3-PRO-V2

### ISSUE THREAD

**Issue by Mkampff** [fr3ak2402/GalaxySlicer-Profile-Library#11]
is posible to add a profile for tronxy xy3-pro-v2 30x30x40 direct bmg capasitive sensor. thanks in advance

**Comment by fr3ak2402** [comment]
@Mkampff 

Yes, I can create a profile for the Tronxy XY3-Pro V2. I will add the printer to the list so that it will be included soon.
